README.md

```diff
@@ -1,2 +1,18 @@
-# daccs-project-website
-Information site about the DACCS project
+# The DACCS Project Website
+This site is to give a summary of what the DACCS project is about.  It includes a mission statement, vision statement, a list of the Principle Investigators and a list of the Funders.
+
+# Building the site
+
+Clone the repository at [https://github.com/DACCS-Climate/daccs-project-website](https://github.com/DACCS-Climate/daccs-project-website)
+
+Navigate to the root directory and build the site by running 
+```angular2html
+python build.py
```

Comment:
`python3` 

`python` is python 2 on most distros

(and same below)

Reply:
Changed to python3